### DISCUSSION TITLE
请问css是如何设置的

### DISCUSSION THREAD

**Discussion by zhuhaisto** [tamlok/vnote#2172] [Ideas]
![QQ截图20220701191058](https://user-images.githubusercontent.com/108032980/176883863-b6229ff5-be93-43af-9c16-c36d74cc66e5.png)
如图 各位大佬如何做css呢

**Comment by tamlok** [comment]
你那个无序列表已经不是子项了，具体参考软件帮助菜单的语法指南。

css看一下官方文档

**Reply by zhuhaisto** [reply]
大佬有用户群吗？qq 微信啥的群

**Reply by zhuhaisto** [reply]
我查询markdown 没看到子项 只看到有序列表和无序列表
![图片](https://user-images.githubusercontent.com/108032980/176928754-d2307d2b-4a88-4c6b-b7eb-eef90a5fbb94.png)
这个也伸缩得太多了吧

**Reply by tamlok** [reply]
> 大佬有用户群吗？qq 微信啥的群

QQ搜一下vnote
